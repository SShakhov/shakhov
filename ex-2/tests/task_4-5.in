my son
Say your prayers, little one. Don't forget, , to include everyone.
44
