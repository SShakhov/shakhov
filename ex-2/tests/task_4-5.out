Say your prayers, little one. Don't forget, my son, to include everyone.
